src/index.tsx

```diff
@@ -126,7 +125,7 @@ const SplitApp = () => {
   const {
     context: { ensService },
   } = useContext();
-  const mainnetProvider = useJsonRpcProvider(DEFAULT_CHAIN_ID);
+  const mainnetProvider = useJsonRpcProvider(1);

```

Comment:
Why no default id?

Reply:
because when you run yarn dev the default chain id is 1337, also this is mainnet provider, it is always id 1